ohmec_data_na.js

```diff
@@ -4,6 +4,146 @@ dataNA = {
   "_comment3":"SPDX-License-Identifier: Apache-2.0",
   "type":"FeatureCollection",
   "features":[
+    { "type":"Feature",
+      "id":"OH700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Native American",
+        "entity2type":"people",
+        "entity2name":"Ohlone",
+        "labelRotate":45,
+        "labelY":0,
+        "labelX":0,
+        "labelScale":2.0,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Ohlone",
+        "startdatestr":"700:01:01",
+        "enddatestr":"1769:01:01"},
+      "geometry":{
+        "_comment1":"Bay Area",
+        "type":"Polygon",
+        "coordinates":[[[-122.5, 38],[-122.5, 36.5],[-120.5, 36.5],[-120.5, 38]]]}},
+    { "type":"Feature",
+      "id":"ES700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Native American",
+        "entity2type":"people",
+        "entity2name":"Esselen",
+        "labelRotate":45,
+        "labelY":0,
+        "labelX":0,
+        "labelScale":2.0,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Esselen",
+        "startdatestr":"700:01:01",
```

Comment:
I imagine there was some movement between those 1000+ years, but I imagine this is about as accurate as we can get at the moment, or at least for starters.

Reply:
These are just placeholder dates for now. These dates will be tightened up over time.